lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -385,6 +391,7 @@ EOSQL
            JOIN editor_subscribe_collection esc ON esc.collection = ${coll_table}.collection
          WHERE esc.available"
     } entities_with('collections'));
+    # FIXME: very similar to $EDIT_IDS_FOR_COLLECTION_SQL, should be generalized
```

Comment:
Should be done ideally, but not now. ;-) Just marking it for future reference (and so that someone grepping for the constant will find this, too).